tracker/interface.go

```diff
@@ -71,6 +72,10 @@ type Interface interface {
 	// so that we are notified for appropriate object changes.
 	OnChanged(obj interface{})
 
+	// GetObservers returns the names of all observers for the given
+	// object.
+	GetObservers(obj interface{}) []types.NamespacedName
+
```

Comment:
I'd drop this from the interface since it breaks anyone using this package

Reply:
How am I supposed to use the added functionality then though? Return the struct type from `New`?

Reply:
> Return the struct type from New?

blah I thought it was already doing that :(